src/net/tracer.rs

```diff
@@ -1,116 +1,15 @@
 use std::{fmt, time::Duration};
 
-use axum::{
-	body::{boxed, Body, BoxBody},
-	extract::MatchedPath,
-	headers::{
-		authorization::{Basic, Bearer},
-		Authorization, Origin,
-	},
-	Extension, RequestPartsExt, TypedHeader,
-};
-use futures_util::future::BoxFuture;
-use http::{header, request::Parts, StatusCode};
+use axum::extract::MatchedPath;
+use http::header;
 use hyper::{Request, Response};
-use surrealdb::{dbs::Session, iam::verify::token};
 use tower_http::{
-	auth::AsyncAuthorizeRequest,
 	request_id::RequestId,
 	trace::{MakeSpan, OnFailure, OnRequest, OnResponse},
 };
 use tracing::{field, Level, Span};
 
-use crate::{dbs::DB, err::Error, iam::verify::basic};
```

Comment:
This code was a leftover. It was moved to `src/net/auth.rs` but not removed from here